packages/server/postgres/migrations/1656522925899_addAzureDevOpsDimensionFieldMapPKConstraint.ts

```diff
@@ -0,0 +1,29 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  ALTER TABLE "AzureDevOpsDimensionFieldMap"
```

Comment:
+1 You could have combined this with the addAzureDevOpsDimensionFieldMapWorkItemType migration

Reply:
Sounds good.  I was going to do that originally, but I didn't know if you wanted a more explicit migration.  I'll modify.

Reply:
Changed